autoload/codefmt.vim

```diff
@@ -437,12 +437,17 @@ function! codefmt#GetYAPFFormatter() abort
     let l:cmd = [l:executable, '--lines=' . a:startline . '-' . a:endline]
     let l:input = join(l:lines, "\n")
 
-    let l:result = maktaba#syscall#Create(l:cmd).WithStdin(l:input).Call(0)
-    let l:formatted = split(l:result.stdout, "\n")
+    try
+      let l:result = maktaba#syscall#Create(l:cmd).WithStdin(l:input).Call()
+      let l:formatted = split(l:result.stdout, "\n")
+
+      let l:full_formatted = l:formatted
```

Comment:
Could you drop this reassignment while you're in here? AFAICT it serves no purpose.